### DISCUSSION TITLE
Plugin Tables

### DISCUSSION THREAD

**Discussion by blanch80** [liedekef/events-made-easy#243] [Q&A]
Hi,

Is there a list of plugin tables anywhere please?

I'm running maintenance on our site which used to run a different events manager plugin a few years ago, I don't want to delete an EME table by mistake.

Do all EME tables start with eme, as we have some starting em that I'm unsure whether can be deleted or not.

Thanks,
David.

**Comment by liedekef** [comment]
all EME tables have your WP database prefix and then "eme_".
If a table has just "em" it is not EME related. See at the top of eme-install.php for all EME database constants (in other words, the table names)

**Reply by blanch80** [reply]
That's great, thanks for confirming.
